runhouse/servers/cluster_servlet.py

```diff
@@ -261,38 +261,41 @@ async def aperiodic_cluster_checks(self):
                     break
 
                 logger.debug("Performing cluster checks")
-                status: ResourceStatusData = await self.astatus()
+                if should_send_status_and_logs_to_den:
+                    # status type: ResourceStatusData
+                    status, send_status_to_den_resp = await self.astatus(
```

Comment:
isn't is possible for `send_status_to_den_resp` to be None here if we're not sending to Den? is that being accounted for below?

Reply:
Yes. If `should_send_status_and_logs_to_den` is false, we don't send the status info to den, and the `send_status_to_den_resp` is set to `None`. Later on, in line 277, we check if `should_send_status_and_logs_to_den` is false, and if this is the case, we break ( = exiting the status check loop). Therefore it is ok if the `send_status_to_den_resp` is None in case `should_send_status_and_logs_to_den` is false.